libbeat/outputs/redis/client.go

```diff
@@ -241,6 +242,8 @@ func (c *client) publishEventsBulk(conn redis.Conn, command string) publishFn {
 
 		}
 
+		took := time.Since(start)
```

Comment:
move to right after Do command, we want to capture time spent even in error conditions.